cmd/ledger/main.go

```diff
@@ -32,7 +32,7 @@ import (
 )
 
 var (
-	accounts    = flag.Int("accounts", 1000000, "the number of accounts")
+	accounts    = flag.Int("accounts", 100, "the number of accounts")
```

Comment:
Why?

Reply:
Nice catch. This is just for local testing...